torch/_inductor/lowering.py

```diff
@@ -4520,6 +4520,14 @@ def fn(*args, **kwargs):
 register_inplace(aten.__ixor__, aten.__xor__)
 
 
+@register_lowering(aten.sym_constrain_range)
+def sym_constrain_range(a, min, max):
```

Comment:
This is not used?

Reply:
@lezcano , I believe this part is needed. Otherwise we get:

```
...
torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised:
MissingOperatorWithoutDecomp: missing lowering
  target: aten.sym_constrain_range.default
```